### ISSUE TITLE
Change wording on "Open in Atom" button

### ISSUE THREAD

**Issue by adamreisnz** [desktop/desktop#8322]
![image](https://user-images.githubusercontent.com/490562/65509003-90f0e080-df25-11e9-8cbc-fd9d8fc13b9a.png)

Looks like this is still an old reference hard coded to Atom, and should instead read "Open with default program".

I no longer have Atom installed, and VS Code is now my default editor, and clicking the button does open VS Code.

This is for the Mac beta version.

**Comment by niik** [comment]
Possibly related to https://github.com/desktop/desktop/issues/8315, cc @kuychaco

**Comment by mislav** [comment]
@adamreisnz Thank you for reporting. When you say “VS Code is now my default editor”, do you mean that it's your default editor on the system level (e.g., default for opening `*.js` files), or that you've set it as your editor from GitHub Desktop under “Preferences…” > “Advanced”?

There are some known issues around detecting that editors available on the system have changed (being installed/uninstalled), but the setting in Preferences might help you manually choose one. One thing you could also check is whether restarting Desktop changes anything re: available editors. Thank you!

**Comment by adamreisnz** [comment]
It's probably irrelevant, I was merely pointing out that the tooltip mentions "Your default editor", whereas in the button it mentions "Atom".

To top it off, when I right click a repo, it mentions VS Code:

![image](https://user-images.githubusercontent.com/490562/65518378-547ab000-df38-11e9-9f11-c06ee9f2be8d.png)

So it seems to me that the button where it says "Atom" it's simply hardcoded and may need to be updated.

And yes, VS Code is my default editor in Github desktop:

![image](https://user-images.githubusercontent.com/490562/65518495-7d02aa00-df38-11e9-829e-13c34d11225d.png)

**Comment by kuychaco** [comment]
Looked into this a bit while fixing #8315. It looks like we're using `resolvedExternalEditor` for the button copy, and we're using `selectedExternalEditor` when the button is clicked. Seems like `resolvedExternalEditor` isn't getting updated properly. 

And the more I dig into this code, the more I think it's worth considering dropping `selectedExternalEditor` because the two seem pretty redundant anyhow. I can't imagine a scenario where the desired state would be for these two values to differ. 

I'm going to hold off on refactoring and investigating this further until we've shipped our onboarding tutorial feature, as that's top priority right now. This incorrect button wording is certainly confusing and irksome, and I've added this issue to our backlog so that I can revisit it later.
